index.html

```diff
@@ -0,0 +1,137 @@
+          <input
+            type="email"
+            name="email"
+            id="email"
+            class="form-control"
+            placeholder="Enter your Email"
+            required
+          />
+        </div>
+        <div class="form-group">
+          <label id="number-label" for="number"
+            >Phone numer</label
+          >
+          <input
+            type="number"
+            name="phone-numer"
+            id="number"
+            min="10"
+            max="99"
+            class="form-control"
+            placeholder="phone number"
+          />
+        </div>
+        <div class="form-group">
+          <p>Lorem ipsum dolorLorem ipsum dolorLorem ipsum dolor</p>
+          <select id="dropdown" name="role" class="form-control" required>
+            <option disabled selected value>Lorem ipsum dolorLorem ipsum dolor</option>
+            <option value="Lorem ipsum dolor">Lorem ipsum dolor</option>
+            <option value="Lorem ipsum dolor">Lorem ipsum dolor</option>
+            <option value="Lorem ipsum dolor">Lorem ipsum dolor</option>
+            <option value="Lorem ipsum dolor">Lorem ipsum dolor</option>
+            <option value="Lorem ipsum dolor">Lorem ipsum dolor</option>
+          </select>
+        </div>
+    
+        <div class="form-group">
+          <p>Lorem ipsum dolorLorem ipsum dolorLorem ipsum dolor</p>
+          <label>
+            <input
+              name="user-recommend"
+              value="definitely"
+              type="radio"
+              class="input-radio"
+              checked
+            />Lorem ipsum dolor</label
+          >
+          <label>
+            <input
+              name="user-recommend"
+              value="maybe"
+              type="radio"
+              class="input-radio"
+            />Lorem ipsum dolor</label
+          >
+        </div>
+    
+        <div class="form-group">
+          <p>
+            Lorem ipsumLorem ipsumLorem ipsum
+          </p>
+    
+          <label
+            ><input
+              name="prefer"
+              value="front-end-projects"
+              type="checkbox"
+              class="input-checkbox"
+            />Lorem ipsum</label
+          >
+          <label>
+            <input
+              name="prefer"
+              value="back-end-projects"
+              type="checkbox"
+              class="input-checkbox"
+            />Lorem ipsum</label
+          >
+        </div>
+    
+        <div class="form-group">
+          <p>your comments here</p>
+          <textarea
+            id="comments"
+            class="input-textarea"
+            placeholder="Lorem ipsumLorem ipsum..."
+          ></textarea>
+        </div>
+    
+        <div class="form-group">
+          <button type="submit" id="submit" class="submit-button">
+            Submit
+          </button>
+        </div>
+      </form>
+    </div>
+    
+
+
+
+    <script  src="js/index.js"></script>
+  </body>
+</html>
```

Comment:
Please kindly avoid : 
1. too much space between lines of code
2. equalize indentation in the code